src/Microsoft.Data.Analysis/DataFrame.IO.cs

```diff
@@ -306,5 +309,123 @@ private static DataFrameColumn CreateColumn(Type kind, string[] columnNames, int
                 return ret;
             }
         }
+
+        /// <summary>
+        /// Writes a DataFrame into a CSV.
+        /// </summary>
+        /// <param name="dataFrame"><see cref="DataFrame"/></param>
+        /// <param name="path">CSV file path</param>
+        /// <param name="separator">column separator</param>
+        /// <param name="header">has a header or not</param>
+        /// <param name="encoding">The character encoding. Defaults to UTF8 if not specified</param>
+        /// <param name="cultureInfo">culture info for handeling decimal values</param>
```

Comment:
```suggestion
        /// <param name="cultureInfo">culture info for formatting values</param>
```

`DateTime` values could be formatted with this culture info as well.